src/PKSim.Core/Snapshots/Mappers/ExtendedPropertyMapper.cs

```diff
@@ -34,20 +35,60 @@ private Task<ModelExtendedProperty> mapExtendedProperty<T>(SnapshotExtendedPrope
             Name = snapshot.Name,
             Value = propertyValue
          };
-         addOptionsToList(snapshot.ListOfValues, option => extendedProperty.AddToListOfValues(convertToTypeFunc(option)));
+         addOptionsToList(snapshot.ListOfValues, option => extendedProperty.AddToListOfValues(convertToTypeFunc(option.ToString())));
          return Task.FromResult<ModelExtendedProperty>(extendedProperty);
       }
 
+      private ExtendedPropertyType mapExtendedPropertyType(Type type)
+      {
+         if (type == typeof(int))
+            return ExtendedPropertyType.Integer;
+
+         if (type == typeof(double))
+            return ExtendedPropertyType.Double;
+
+         if (type == typeof(bool))
+            return ExtendedPropertyType.Boolean;
+
+         return ExtendedPropertyType.String;
+      }
+
       public override Task<ModelExtendedProperty> MapToModel(SnapshotExtendedProperty snapshot, SnapshotContext snapshotContext)
       {
-         if (double.TryParse(snapshot.Value.ToString(), out var doubleResult))
-            return mapExtendedProperty(snapshot, option => double.Parse(option.ToString()), doubleResult);
```

Comment:
@Yuri05 This was the reason. 7E3 is a valid string as a double. Naming a molecule like a double is completely crazy so that's why it was assumed to be ok. But clearly not lol